pygotham/talks/models.py

```diff
@@ -20,6 +20,22 @@ def __str__(self):
         return self.name
 
 
+class Duration(db.Model):
+
+    """Talk duration."""
+
+    __tablename__ = 'durations'
+
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(255), nullable=False)
+    duration = db.Column(db.Integer, nullable=False)
+    inactive = db.Column(db.Boolean, default=False)
```

Comment:
I agree. In this case, however, I want things to be active unless explicitly set otherwise so I figured specifying it the other way around made sense.